test/e2e/specs/editor/various/block-editor-keyboard-shortcuts.spec.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+const MatchObject = `<!-- wp:paragraph -->
+<p>1st</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>2nd</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>3rd</p>
+<!-- /wp:paragraph -->`;
+
+test.describe( 'block editor keyboard shortcuts', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'should move the single block up', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await page.getByRole( 'button', { name: 'Add default block' } ).click();
+		await page.keyboard.type( '1st' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '2nd' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '3rd' );
+		await expect.poll( editor.getEditedPostContent ).toBe( MatchObject );
+		await pageUtils.pressKeys( 'secondary', 't' );
+		await pageUtils.pressKeys( 'secondary', 'y' );
+		await expect.poll( editor.getEditedPostContent ).toBe( MatchObject );
+	} );
+
+	test( 'should move the single block down', async ( { editor, page } ) => {
+		await page.getByRole( 'button', { name: 'Add default block' } ).click();
+		await page.keyboard.type( '1st' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '2nd' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '3rd' );
+		await expect.poll( editor.getEditedPostContent ).toBe( MatchObject );
+		await page.keyboard.press( 'ArrowUp' );
+		await expect.poll( editor.getEditedPostContent ).toBe( MatchObject );
+	} );
+
+	test( 'should move the multiple block up', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await page.getByRole( 'button', { name: 'Add default block' } ).click();
+		await page.keyboard.type( '1st' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '2nd' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '3rd' );
+		await expect.poll( editor.getEditedPostContent ).toBe( MatchObject );
+		await page.keyboard.press( 'ArrowUp' );
+		page.locator( '[aria-label="Multiple selected blocks"]' );
```

Comment:
Call the just the locator doesn't do anything. This should be converted into a proper assertion.